etna/transforms/trend.py

```diff
@@ -0,0 +1,182 @@
+from typing import Dict
+from typing import List
+from typing import Optional
+
+import pandas as pd
+from ruptures.base import BaseCost
+from sklearn.linear_model import LinearRegression
+
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.binseg import _Binseg
+from etna.transforms.change_points_trend import BaseEstimator
+from etna.transforms.change_points_trend import TDetrendModel
+from etna.transforms.change_points_trend import TTimestampInterval
+from etna.transforms.change_points_trend import _OneSegmentChangePointsTrendTransform
+
+
+class _OneSegmentTrendTransform(_OneSegmentChangePointsTrendTransform):
+    """_OneSegmentTrendTransform adds trend as a feature. Creates column 'regressor_<in_column>_trend'."""
+
+    def __init__(
+        self,
+        in_column: str,
+        change_point_model: BaseEstimator,
+        detrend_model: TDetrendModel,
+        **change_point_model_predict_params,
+    ):
+        """Init _OneSegmentTrendTransform.
+
+        Parameters
+        ----------
+        in_column:
+            name of column to apply transform to
+        change_point_model:
+            model to get trend change points
+        detrend_model:
+            model to get trend from data
+        change_point_model_predict_params:
+            params for change_point_model predict method
+        """
+        self.in_column = in_column
+        self.out_column = "regressor_" + in_column + "_trend"
+        self.change_point_model = change_point_model
+        self.detrend_model = detrend_model
+        self.per_interval_models: Optional[Dict[TTimestampInterval, TDetrendModel]] = None
+        self.intervals: Optional[List[TTimestampInterval]] = None
+        self.change_point_model_predict_params = change_point_model_predict_params
+        super().__init__(
+            in_column=self.in_column,
+            change_point_model=self.change_point_model,
+            detrend_model=self.detrend_model,
+            **self.change_point_model_predict_params,
+        )
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Add column 'regressor_<in_column>_trend' with trend, got from the detrend_model.
+
+        Parameters
+        ----------
+        df:
+            data to get trend from
+
+        Returns
+        -------
+        pd.DataFrame:
```

Comment:
check please that docs looks good with this Return section

how to build the docs: https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md#documentation